docs/03-tutorials/00-application-connectivity/ac-04-register-secured-api.md

```diff
@@ -38,11 +38,11 @@ The **entries** object must contain the following fields:
 
 The **credentials** object must contain the following fields:
 
-| Field                 | Description                                                  |
-| --------------------- | ------------------------------------------------------------ |
-| **secretName**        | Name of a Secret storing credentials                         |
-| **type**              | Authentication method type. Supported values: `Basic`, `OAuth`, `CertGen`. |
-| **authenticationUrl** | Optional OAuth token URL, valid only for the OAuth type.           |
+| Field                 | Description                                                                 |
+| --------------------- |-----------------------------------------------------------------------------|
+| **secretName**        | Name of a Secret storing credentials                                        |
+| **type**              | Authentication method type. Supported values: `Basic`, `OAuth`, `CertGen`.  |
```

Comment:
This 
![image](https://user-images.githubusercontent.com/53601578/172589504-fe965cee-a3a3-4a0e-b888-f2b3a1995795.png)
suggests that we should also add another supported type here, am I right? 
```suggestion
| **type**              | Authentication method type. Supported values: `Basic`, `OAuth`, `OAuthWithCert `, `CertGen`.  |
```

Reply:
That's correct!